src/shared/types/Approach.ts

```diff
@@ -2,26 +2,27 @@ import { DatabaseItem, ProcedureTransition } from './Common';
 import { ProcedureLeg } from './ProcedureLeg';
 
 export enum ApproachType {
-    Unknown,
-    LocBackcourse,
-    VorDme,
-    Fms,
-    Igs,
-    Ils,
-    Gls,
-    Loc,
-    Mls,
-    Ndb,
-    Gps,
-    NdbDme,
-    Rnav,
-    Vortac,
-    Tacan,
-    Sdf,
-    Vor,
-    MlsTypeA,
-    Lda,
-    MlsTypeBC,
+    Unknown = 'UKN',
+    LocBackcourse = 'LOCBC',
+    VorDme = 'VDME',
+    Fms = 'FMS',
+    Igs = 'IGS',
+    Ils = 'ILS',
+    Gls = 'GLS',
+    Loc = 'LOC',
+    Mls = 'MLS',
+    Ndb = 'NDB',
+    Gps = 'GPS',
+    NdbDme = 'NDME',
+    Rnav = 'RNV',
+    Vortac = 'VTAC',
+    Tacan = 'TCN',
+    Sdf = 'SDN',
+    Vor = 'VOR',
+    MlsTypeA = 'MLSA',
+    Lda = 'LDA',
+    MlsTypeBC = 'MLSBC',
+    Visual = 'VIS',
```

Comment:
Should probably note in a JSDoc that this is for FMS implementation convenience and not from ARINC424